HISTORY.rst

```diff
@@ -3,6 +3,14 @@
 History
 -------
 
+1.2.3 (2017-01-24)
```

Comment:
Can you remove all this unrelated release stuff? By convention it is committed directly on the branch and that commit is than tagged. I think merging such a commit might be messy.